jep/0000/README.adoc

```diff
@@ -0,0 +1,338 @@
+
+| Discussions-To
+| https://community.jenkins.io/t/jenkins-terminology-cleanup-initiative-coordination/180
+
+
+|===
+
+== Abstract
+
+The goal of this JEP is to have the latest status of deprecated word or terms and agreed replacements in a table at one place that can be discussed with history.
+The proposal is to have git to keep history of changes, and discussion on new term on PRs.
+
+Deprecated term are written explicitly to be searchable.
+
+== Specification
+
+* The term in table must be in lower case (when the language allows it). 
+
+* To add a new language, propose a PR on top of this JEP by a copy/paste one language (other than English) and add a new section.
+
+* To add a new term in an existing language, propose a PR on top of this JEP with the modified table.
+
+TBD: stands for "To Be Done", it means that there is no community agreement yet. If you have a proposal feel welcome to propose it as a PR or in https://community.jenkins.io/t/jenkins-terminology-cleanup-initiative-coordination/180[Discourse terminology thread]
+
+=== English - Default
+
+* property suffix: <none>
+
+[options="header"]
+|======================
+| Deprecated term root           | New term root                 | New term declinaisons and comments              
+| slave                          | agent                         | agents (plural)                                 
+| master                         | controller                    | Frontend for Jenkins users and administrators.                     
+| master                         | built-in node                 | Jenkins “main node” label - Aligned with “Jenkins as node”
+| whitelist                      | allowlist                     | default but can be adapted to context           
+| blacklist                      | denylist or blocklist         | default but can be adapted to context           
+|======================
+
+=== Translations
+
+==== French
+
+* property suffix: fr
+
+* Table
+
+[options="header"]
+|======================
+| English term                  |Deprecated term                | New term root                 | Comments                                                 
+| agent                         | esclave, esclaves             | agent                         | "esclave" can be valid in a context of a spoken language.
+| controller                    | maître, maitre                | contrôleur                    |                                                          
+| built-in node                 | maître, maitre                | TDB                           |                                                          
+| allowlist                     |                               |                               |                                                          
+| denylist or blocklist         |                               |                               |                                                          
```

Comment:
IIRC the consensus was to avoid `blocklist` in our codebase and terms. CC @halkeye @uhafner to confirm

```suggestion
| denylist         |                               |                               |                                                          
```